js/abp-preprocessed.es6.js

```diff
@@ -25,16 +25,18 @@ easylists = {
  */
 for (let list in easylists) {
     let url = easylists[list].url;
-    let listData = load.loadExtensionFile(url, '', 'external');
+    load.loadExtensionFile(url, '', 'external', (listData) => {
     let whitelistFile = easylists[list].whitelist;
 
-    // append the whitelist entries before we process the list
-    if (whitelistFile) {
-        let whitelist = load.loadExtensionFile(whitelistFile);
-        listData += whitelist;
-    }
-    
-    abp.parse(listData, easylists[list].parsed);
+        // append the whitelist entries before we process the list
+        if (whitelistFile) {
+            load.loadExtensionFile(whitelistFile, '', '', (whitelist) => {
+                listData += whitelist
+                abp.parse(listData, easylists[list].parsed)
+            })
+        } else {
+            abp.parse(listData, easylists[list].parsed)
+        }
+    });
 }
-
 easylists.loaded = true;
```

Comment:
Thanks, I forgot about this. 

I added a "loaded" value for each list type here:
https://github.com/duckduckgo/chrome-zeroclickinfo/pull/156/files#diff-b5e2ac6ae5c6ea4e1b169eb86312c3a6R36

And checking that value in trackers.js
https://github.com/duckduckgo/chrome-zeroclickinfo/pull/156/files#diff-572667045eedadf469db0620588c4c47R82